js/main.js

```diff
@@ -0,0 +1,33 @@
+var $image = document.querySelector('img');
+var $photoUrl = document.querySelector('.photo-url');
+var $formEntry = document.querySelector('form');
+var $titleInput = document.getElementsByName('title');
+var $photoInput = document.getElementsByName('photo-URL');
+var $notesInput = document.getElementsByName('notes');
```

Comment:
You should just give these elements a class so that you can use `querySelector` easily.